src/backend/auth/firebot-device-auth-provider.ts

```diff
@@ -1,5 +1,5 @@
 import logger from "../logwrapper";
-import accountAccess, { FirebotAccount } from "../common/account-access";
+import accountAccess, { FirebotAccount, setBotTokenIssue, setStreamerTokenIssue } from "../common/account-access";
```

Comment:
You shouldn't need to import any additional functions here. The default `accountAccess` object should contain what you need.